src/utils/constants/constants.js

```diff
@@ -5,16 +5,4 @@ export const STATUS = {
   GREEN: 3
 }
 
-export const DEFAULT_CRITERIA = [
-  STATUS.NOT_SET,
-  STATUS.NOT_SET,
-  STATUS.NOT_SET,
-  STATUS.NOT_SET,
-  STATUS.NOT_SET,
-  STATUS.NOT_SET,
-  STATUS.NOT_SET,
-  STATUS.NOT_SET,
-  STATUS.NOT_SET,
-  STATUS.NOT_SET,
-  STATUS.NOT_SET
-]
+export const DEFAULT_CRITERIA = Array(10).fill(0);
```

Comment:
Whoever found something this cool should be given a promotion... 😉

Reply:
Haha. Oops, I'll change this